### ISSUE TITLE
NgRx 8 createAction syntax

### ISSUE THREAD

**Issue by alecrt** [xipheCom/ngrx-batch-action-reducer#5]
Hi @xipheCom,

any idea on how to use this module with new NgRx 8 "createAction" syntax?

Thank you,
AC

**Comment by progral** [comment]
Nope. If you really need it, you can still use the "old" syntax with ngrx.

**Comment by krambono** [comment]
I would say thank you to the author ! I ran into a similar problem with normalized store and hopefully I saw what you did.
We can't use the new syntax I guess. 
I wrote that :
```
export class MultiAction implements Action {
  readonly type = 'Multi';
  public isBatchAction = true;
  constructor(public payload: Action[]) {}
}
```
And it's working perfectly ! I don't use the decorator, I had ts errors with it.
